src/tests/Common/CLRTest.NativeAot.targets

```diff
@@ -152,7 +152,7 @@ if defined RunNativeAot (
     <OutputType>Exe</OutputType>
     <OutputPath>%24(MSBuildProjectDirectory)\</OutputPath>
     <IntermediateOutputPath>%24(MSBuildProjectDirectory)\</IntermediateOutputPath>
-    <PlatformTarget>$(TargetArchitecture)</PlatformTarget>
+    <TargetArchitecture>$(TargetArchitecture)</TargetArchitecture>
```

Comment:
```suggestion
```

Reply:
Are you sure it is not needed?  That is the content of the `CDATA` section, which will be written to the new project file with `$(TargetArchitecture)` replaced with its actual value. For instance, the next line in that `CDATA` section is:
```xml
<Optimize>$(Optimize)</Optimize>
```

Reply:
Ah ok. I have not looked at the context. Ignore my comment then.